queries/injections.scm

```diff
@@ -7,7 +7,9 @@
     (member_expression
       property: (property_identifier) @injection.language)
   ]
-  arguments: (template_string) @injection.content)
+  arguments: (template_string
+    (template_chars) @injection.content
```

Comment:
`test/highlight/injection.js` was failing, which tests that for the source text:
```
eval(js `var foo`)
```
the `var foo` gets highlighted as JS content (eg `var` gets highlighted as a keyword)

My understanding is that the reason that test started failing is because the way that injections work, by default this `@injection.content` doesn't include ("re-parse") any child nodes' content of this `(template_string)` node. And now there is a `(template_chars)` child node, so the re-parsing of the `(template_string)` node doesn't include the `var foo` content

So this gets that test passing by specifying that the new `(template_chars)` node should be used as the thing to be re-parsed instead

I have no idea if this is hacky or not, eg not sure how the old version vs this version would behave when highlighting eg a template string with multiple text sections (`` eval(`is this text parsed as JS ${foo} and then this text is also parsed as JS`) ``)